### PR TITLE
Initial docs reworked to Squire

### PR DESCRIPTION
**Pull request by baell**
Our docs are ready with a few cleanup items, like images & brand style, to swap out.

{{@echolayer summary}}

{{@echolayer changes}}

### CODE REVIEW

File: introduction.mdx [hunk 1]
```diff
@@ -14,58 +14,48 @@ description: 'Welcome to the home of your new documentation'
...
+First you'll need to create an account and authorize with Squire:
 
-<CardGroup cols={2}>
+<CardGroup cols={1}>
   <Card
-    title="Edit Your Docs"
+    title="Sign up"
     icon="pen-to-square"
-    href="https://mintlify.com/docs/quickstart"
+    href="https://app.echolayer.com/sign-in"
```
Comment (karlclement) [inline introduction.mdx]: You can now use app.squire.ai instead of app.echolayer.com

File: introduction.mdx [hunk 2]
```diff
@@ -14,58 +14,48 @@ description: 'Welcome to the home of your new documentation'
...
-    href="https://mintlify.com/docs/development"
-  >
-    Preview your changes before you push to make sure they're perfect
+    Get your account and authorize with GitHub
   </Card>
 </CardGroup>
 
-## Make it yours
+<Note>
+Today squire has a constraint of working with Typescript, including react, repositories which allows our team to ensure accuracy in code changes, please select a repository with typescript in order to get full value today.
```
Comment (karlclement) [inline introduction.mdx]: Capitalize Squire here

File: mint.json [hunk 3]
```diff
@@ -18,74 +18,45 @@
...
       "name": "Support",
-      "url": "mailto:hi@mintlify.com"
+      "url": "mailto:support@squire.ai"
     }
   ],
   "topbarCtaButton": {
-    "name": "Dashboard",
-    "url": "https://dashboard.mintlify.com"
+    "name": "Get Started",
+    "url": "https://app.echolayer.com/sign-in"
```
Comment (karlclement) [inline mint.json]: Same here

**Review by karlclement — APPROVED** [review]

File: README.md
Comment (karlclement) [inline README.md:1]: Id also leave the original README if it doesnt affect the docs, that way we know how to set it up

Comment (baell) [reply README.md:1]: This is a good point, my rational for changing was people might go look at our public repo.

I truly am good either way, what's your preference?

**Comment by baell** [comment]
@karlclement Addressed all other feedback in: https://github.com/echolayer/squire-docs/pull/1/commits/9d7edac061cc142559bd749a0de28aec19387ff4

**Pull request merged**